xla/tests/onednn_matmul_test.cc

```diff
@@ -923,6 +951,117 @@ TEST_F(MatmulTest, TestF32ConstantWeights) {
   )");
 }
 
+// MM + BiasAdd + Elu : FP32
+TEST_F(MatmulTest, BiasAddELUFusion_F32) {
+  const char* matmul_module_str = R"(
+  HloModule matmul.test.f32
+
```

Comment:
Can you please clean this up a bit?

- sort, remove unnecessary attributes
- fix names
- remove default layouts

Reply:
done. @jreiffers can you please take a relook?

Reply:
Thank you very much!